lib/screen_controller.dart

```diff
@@ -386,11 +386,14 @@ class ScreenController {
       }
     } else if (action is CopyToClipboardAction) {
       if (action.value != null) {
-        Clipboard.setData(ClipboardData(text: action.value!)).then((value) {
-          if (action.onSuccess != null) {
-            executeAction(context, action.onSuccess!);
-          }
-        });
+        String? clipboardValue = dataContext.eval(action.value);
```

Comment:
can we abstract out the eval and do it inside the Action. See StartTimerAction's `getStartAfter()`

Reply:
Refactored it